REFERENCE.md

```diff
@@ -26,6 +26,10 @@ The following parameters are available in the `patroni` class:
 * [`scope`](#scope)
 * [`namespace`](#namespace)
 * [`hostname`](#hostname)
+* [`is_standbyl`](#is_standby)
```

Comment:
No need to run `rake reference` and commit this as it will happen during CI.

Reply:
sure , will remove these changes.thanks.